benchbuild/utils/container.py

```diff
@@ -57,13 +57,32 @@ def get_container_url():
 
 
 def is_valid_container(path):
-    return False
-
+    try:
+        tmp_hash_path = __CONTAINER_DEFAULT__ + ".hash" 
+        tmp_hash_file = open(tmp_hash_path, 'r')
+        tmp_hash = tmp_hash_file.readline() 
+    except:
+        print("No .hash-file in the tmp-directory.")
```

Comment:
No need to bother the user with a print. You got an exception, the container is not valid. Log the error via 'logging' and return false.